internal/installation/context_windows.go

```diff
@@ -0,0 +1,57 @@
+package installation
+
+import (
+	"fmt"
+	"os/user"
+	"strconv"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/osutils"
+)
+
+const (
+	// adminInstallRegistry is the registry key name used to determine if the State Tool was installed as administrator
+	adminInstallRegistry = "Installed As Admin"
+
+	installRegistryKeyPath = `SOFTWARE\ActiveState\install`
+)
+
+func SaveContext(context *Context) error {
+	user, err := user.Current()
+	if err != nil {
+		return errs.Wrap(err, "Could not get current user")
+	}
+
+	key, _, err := osutils.CreateUserKey(fmt.Sprintf(`%s\%s`, user.Uid, installRegistryKeyPath))
+	if err != nil {
+		return errs.Wrap(err, "Could not create registry key")
+	}
+	defer key.Close()
+
+	err = key.SetStringValue(adminInstallRegistry, strconv.FormatBool(context.InstalledAsAdmin))
+	if err != nil {
+		return errs.Wrap(err, "Could not set registry key value")
+	}
+
+	return nil
+}
+
+func getAdminInstallInformation() (bool, error) {
```

Comment:
As you noted this will error if the key doesn't exist. I think we should return an identifiable error for this and have the consumer check for this, and if the key was not found - assume we installed as user.